modules/imports/src/test/scala/org/dhallj/imports/ImportResolutionSuite.scala

```diff
@@ -58,6 +65,16 @@ class ImportResolutionSuite extends FunSuite {
     assert(resolve(expr) == expected)
   }
 
+  //TODO - dependent on https://github.com/travisbrown/dhallj/issues/34
```

Comment:
I think we can merge this for now as it is here. I'll look into failing more reasonably in the parser on quoted URL paths this weekend.